From 5d93468757bb75e9d8b29599cda0decc124cedb8 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Mon, 28 Feb 2022 09:49:58 +0100
Subject: [PATCH] com.webos.service.mediaindexer.role.json.in: Add access to
 com.webos.app.*

Fixes:

Feb 28 08:28:41 qemux86-64 ls-hubd[239]: [] [pmlog] ls-hubd LSHUB_NO_OUT_PERMS {"DEST_APP_ID":"com.webos.app.imageviewer-87783","SRC_APP_ID":"com.webos.service.mediaindexer","EXE":"/usr/sbin/com.webos.service.mediaindexer","PID":851} "com.webos.service.mediaindexer" does not have sufficient outbound permissions to communicate with "com.webos.app.imageviewer-87783" (cmdline: /usr/sbin/com.webos.service.mediaindexer)

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 files/sysbus/com.webos.service.mediaindexer.role.json.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.mediaindexer.role.json.in b/files/sysbus/com.webos.service.mediaindexer.role.json.in
index 4e072b0..a43f647 100644
--- a/files/sysbus/com.webos.service.mediaindexer.role.json.in
+++ b/files/sysbus/com.webos.service.mediaindexer.role.json.in
@@ -14,7 +14,8 @@
                 "com.webos.service.pdm",
                 "com.webos.mediadb",
                 "com.webos.settingsservice",
-                "com.webos.service.mediaindexer.client*"
+                "com.webos.service.mediaindexer.client*",
+                "com.webos.app.*"
             ]
         },
         {
